hs_core/models.py

```diff
@@ -2206,8 +2231,16 @@ def parse_citation_name(self, name, first_author=False):
 
         return author_name + ", "
 
+    def get_custom_citation(self):
+        """Get custom citation"""
+        try:
+            return str(self.metadata.citation.all()[0]).strip()
+        except:
+            pass
```

Comment:
Log or handle this?  What is the reason for the try/except?  Only thing I see is if the there are no citations?